src/main/kx/c.java

```diff
@@ -36,6 +36,7 @@ public class c {
     int j;
     int J;
     boolean a;
+    boolean cancelled = false;
```

Comment:
this should be `volatile` as we're accessing this variable from 2 different threads (main swing thread when doing cancel and swing background worker)

Reply:
Fixed.